base_contact/__init__.py

```diff
@@ -19,4 +20,7 @@
 #
 ##############################################################################
 
-from . import base_contact
+import logging
+
+
+logging.getLogger(__name__).warn("This module is DEPRECATED. See README.")
```

Comment:
That this will flash every time a server is launched even if the module isn't installed.